build/config/PublishData.json

```diff
@@ -49,6 +49,13 @@
             "vsix": [ "https://dotnet.myget.org/F/roslyn-nonnull/vsix/upload" ],
             "channels": [ "nonnull" ]
         }
```

Comment:
You need a comma following this close `}`

Reply:
Thanks, addressed.